apps/jobs-processing-service/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@novu/jobs-processing-service",
```

Comment:
called it `jobs-processing-service`, but I'm open to other suggestions here

Reply:
@LetItRock I think 'worker-service' may be a better name.

Reply:
@Cliftonz I was wondering if the name shouldn't be more specific to the workflow but at the same time I don't want to have it long name...

Reply:
No because our "workers" can and will be doing many different things, thus I do not think jobs-processin is not the best name.